pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -102,7 +102,7 @@ def advance(self, batch: Any, batch_idx: int) -> None:  # type: ignore[override]
             )
 
             # manually capture logged metrics
-            training_step_output = self.trainer._call_accelerator_hook("training_step", *step_kwargs.values())
+            training_step_output = self.trainer._call_ttp_hook("training_step", *step_kwargs.values())
```

Comment:
Should we use an intermediate `_training_step` which contains:

```py
def _training_step(...):
      with self.precision_plugin.train_step_context():
               self.training_step(...)
```

I am not a huge fan of duplicating the logic within all *_step methods.

Reply:
We have a issue for simplify precision_plugins's X_steps #10891
But here, different plugins has different behaviors of call model(arg,kwarg) or model.train/test/_step(), any idea how can we simplify strategies's x_steps?